doc/kb_sdk_make_ui.md

```diff
@@ -0,0 +1,169 @@
+# <A NAME="top"></A>![alt text](https://avatars2.githubusercontent.com/u/1263946?v=3&s=84 "KBase") [KBase SDK](../README.md)
+
+1. [Install SDK Dependencies](kb_sdk_dependencies.md)
+2. [Install SDK with Docker](kb_sdk_dockerized_install.md)
+3. [Create Module](kb_sdk_create_module.md)
+4. [Specify Module and Method(s)](kb_sdk_edit_module.md)
+5. [Implement Method(s)](kb_sdk_impl_methods.md)
+6. **Specify User Interface**
+7. [Locally Test Module and Method(s)](kb_sdk_local_test_module.md)
+8. [Register Module](kb_sdk_register_module.md)
+9. [Test in KBase](kb_sdk_test_in_kbase.md)
+10. [Complete Module Info](kb_sdk_complete_module_info.md)
+11. [Deploy](kb_sdk_deploy.md)
+
+### <A NAME="ui-widget"></A>6. Specify User Interface
+
+Control of Narrative interaction is accomplished in files in the ui/narrative/methods/<MyMethod> directory.
```

Comment:
The angle brackets will get treated like HTML by markdown and not rendered. so it'll look like `/ui/narrative/methods/` instead. Try using `&lt;` and `&gt;` to escape them.